apps/heft/src/plugins/CopyStaticAssetsPlugin.ts

```diff
@@ -36,16 +36,19 @@ export interface ISharedCopyStaticAssetsConfiguration {
    * Globs that should be explicitly included.
    */
   includeGlobs?: string[];
-}
 
-interface ICopyStaticAssetsConfiguration extends ISharedCopyStaticAssetsConfiguration {
+  /**
+   * The folder to which assets should be copied. For example "lib". This defaults to "lib"
+   */
+  destinationFolderName?: string;
```

Comment:
Should this be a `string[]` of folder names to allow multiple destinations? See my [other comment here](https://github.com/microsoft/rushstack/pull/2315/files#r514448139).

Reply:
Same thing as the other comment